pyrtl/rtllib/pyrtlfloat/_add_sub.py

```diff
@@ -0,0 +1,342 @@
+import pyrtl
+
+from ._float_utills import FloatUtils
+from ._types import PyrtlFloatConfig, RoundingMode
+
+
+class AddSubHelper:
+    @staticmethod
+    def add(
+        config: PyrtlFloatConfig,
+        operand_a: pyrtl.WireVector,
+        operand_b: pyrtl.WireVector,
+    ) -> pyrtl.WireVector:
+        fp_type_props = config.fp_type_properties
```

Comment:
All user-facing methods/functions should check (somewhere, not necessarily here) that the operand bitwidths are consistent with `config`